luzer/counters.c

```diff
@@ -22,15 +22,15 @@ void __sanitizer_cov_pcs_init(uint8_t* pcs_beg, uint8_t* pcs_end);
 } /* extern "C" */
 #endif
 
-static const int kDefaultNumCounters = 1 << 20;
+static const size_t kDefaultNumCounters = 1 << 20;
 
 // Number of counters requested by Lua instrumentation.
-int counter_index = 0;
+size_t counter_index = 0;
 // Number of counters given to Libfuzzer.
```

Comment:
I would split this commit to a number of commits:
- change datatype `int` -> `size_t`
- fix #12
- patch that adds `NO_SANITIZE`
- ...